device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/MultiplexingClient.java

```diff
@@ -460,13 +460,6 @@ public void registerDeviceClients(Iterable<DeviceClient> deviceClients, long tim
                 devicesToRegisterMap.put(deviceClientToRegister.getConfig().getDeviceId(), deviceClientToRegister);
                 DeviceClientConfig configToAdd = deviceClientToRegister.getConfig();
 
-                // Overwrite the sslContext of the new client to match the multiplexing client
-                // settings if it was set. If it wasn't set, defer to the default client to set this ssl context
-                if (this.sslContext != null)
-                {
-                    configToAdd.getAuthenticationProvider().setSSLContext(this.sslContext);
```

Comment:
When multiplexing, we never check the SSLContext of each registered device, so there was never any reason for us to assign a device's SSLContext to match the MultiplexingClient's SSLContext here

Reply:
Where is the SSLContext that is used for Multiplexing scenarios?

Reply:
It is created at constructor time and saved as a local field in this class